### PR TITLE
Add bcm2835 library

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by saraedum** [comment]
This library is probably not terribly useful on anything but a RaspberryPi. Is there some way to build a staged-recipe on aarch64?

**Comment by saraedum** [comment]
@conda-forge/help-c-cpp, this one is ready for review :)

File: recipes/bcm2835/build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+set -ex
+
+export CPPFLAGS="-I$PREFIX/include $CPPFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
+export CFLAGS="-g -O2 $CFLAGS"
```
Comment (xhochy) [inline recipes/bcm2835/build.sh:6]: These flags should all be included in the default ones, except maybe `LD_LIBRARY_PATH` but that shouldn't be needed normally at all.

**Comment by xhochy** [comment]
> This library is probably not terribly useful on anything but a RaspberryPi. Is there some way to build a staged-recipe on aarch64?

No, not yet.

File: recipes/bcm2835/build.sh (hunk 1 above)
Comment (saraedum) [reply recipes/bcm2835/build.sh]: I copied this from elsewhere of course but it's good to know that we do not have to set these explicitly :)

**Comment by saraedum** [comment]
Thanks a lot @xhochy!

**Pull request merged**